src/components/edit/Interactions.jsx

```diff
@@ -155,20 +160,45 @@ export default class EditInteractions extends Component {
     const plantPartToAdd = e.target.value;
     const hiSymptoms = [...this.state.hiSymptoms];
     const plantParts = [...this.state.plantParts];
+    const existingPlantPart = hiSymptoms.find(hiSymptom => hiSymptom.plantPart === plantPartToAdd);
+    const plantPartExists = existingPlantPart ? true : false;
+    const indirect = existingPlantPart ? !existingPlantPart.isIndirect : false;
     const baseHiSymptom = {
```

Comment:
this file is huge, can we move this and the blank host interaction into a constants file?